tracker/telegram/bot.py

```diff
@@ -118,10 +118,9 @@ def escape_html(text: str) -> str:
     Escapes HTML symbols in the text to ensure proper rendering in Telegram messages.
 
     :param text: The input string that may contain HTML symbols.
-    :return: A string with HTML symbols escaped, replacing '&' with '&amp;', '<' with '&lt;',
-             and '>' with '&gt;'.
+    :return: A string with HTML symbols escaped using the Text class.
     """
-    return text.replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;")
+    return Text(text).as_html()
```

Comment:
Please use `html.unparse()` as described in issue

Reply:
I searched for html.unparse() in aiogram's recent official docs, is why I went with this route. Will make requested changes.

Reply:
@emarc99 
```
from aiogram import html

code = "if  a>b: return a<<1;"
print(html.unparse(code))
```